### ISSUE TITLE
Undefined variable $lang (View: /vendor/knuckleswtf/scribe/resources/views/themes/default/index.blade.php)

### ISSUE THREAD

**Issue by Hasban-Fardani** [knuckleswtf/scribe#918]
### Scribe version

4.38.0

### PHP version

8.3.3-1+ubuntu22.04.1+deb.sury.org+1

### Framework

Laravel

### Framework version

11.32.0

### Scribe config

```ruby
auth.enabled => true
auth.default => true
```

### What happened?

I ran `php artisan scribe:`. And then i got this error 'Undefined variable $lang'. 

❯ php artisan scribe:generate
ⓘ Extracting intro and auth Markdown files to: .scribe
✔ Extracted intro and auth Markdown files to: .scribe
ⓘ Writing HTML docs...

   Illuminate\View\ViewException 

  Undefined variable $lang (View: /home/hasban/Code/Github/sarpras-sebelas/backend/vendor/knuckleswtf/scribe/resources/views/themes/default/index.blade.php)

  at storage/framework/views/0f06caf050fa62a6db9b602a044525eb.php:68
     64▕         <div class="dark-box">
     65▕             @if (isset($metadata['example_languages']))
     66▕                 <div class="lang-selector">
     67▕                     @foreach ($metadata['example_languages'] ?? [] as $name => $lang)
  ➜  68▕                         <?php $name = $name ?? $lang; ?>
     69▕                         <button type="button" class="lang-button"
     70▕                             data-language-name="{{ $lang }}">{{ $name }}</button>
     71▕                     @endforeach
     72▕                 </div>

      +1 vendor frames 

  2   storage/framework/views/0f06caf050fa62a6db9b602a044525eb.php:68
      ErrorException::("Undefined variable $lang")

  3   storage/framework/views/0f06caf050fa62a6db9b602a044525eb.php:68
      Illuminate\Foundation\Bootstrap\HandleExceptions::Illuminate\Foundation\Bootstrap\{closure}()

I have also have cleared the config and view cache before, I have deleted the vendor and reinstalled the library. I have not changed anything in the 'example_languages' config. But i still get this error. Why is this happening? 

![image](https://github.com/user-attachments/assets/d619bae4-5523-4a42-b30b-86e4d4067f3f)

### Docs

- [X] I've checked [the docs](https://scribe.knuckles.wtf/laravel), the [troubleshooting guide](https://scribe.knuckles.wtf/laravel/troubleshooting), and [existing issues](https://github.com/knuckleswtf/scribe/issues?q=is%3Aissue+), but I didn't find a solution

**Comment by shalvah** [comment]
Uh...Why does your screenshot have a different error and stack trace from your pasted error message?

Hmm, I can see that the Blade template was not properly compiled to PHP. In the compiled PHP file (`storage/framework/views/<long_name>.php`), you still have `@foreach`, but it should have been replaced with `<?PHP foreach() ?>`. I have no idea why that is happening, though. And you say you've cleared views cache. Could you try deleting the `storage/framework` folder manually?

**Comment by Hasban-Fardani** [comment]
@shalvah sorry I forgot why the error is different :). I tried to delete the storage/framework manually but the error persisted. But after I tried it on other laravel projects the error was the same, and not just for scribe. I tried to run all other laravel projects (without scribe) and still error and error from cache, maybe there is something wrong with my php or laravel config on my linux mint device. I'll reinstall everything or maybe even try reinstalling my linux.
